api/server/router/container/container_routes.go

```diff
@@ -597,6 +597,11 @@ func (s *containerRouter) postContainersCreate(ctx context.Context, w http.Respo
 					return errdefs.InvalidParameter(errors.New("BindOptions.ReadOnlyForceRecursive needs API v1.44 or newer"))
 				}
 			}
+			if m.VolumeOptions != nil {
+				if m.VolumeOptions.SubPath != "" {
+					return errdefs.InvalidParameter(errors.New("VolumeOptions.SubPath needs API v1.44 or newer"))
+				}
+			}
```

Comment:
I wonder if this one should be an error or just be reset on older API versions. I know we reset such things for most (I guess the recursive-readonly was an exception because it was from a security perspective, and ignoring it would weaken security)

Reply:
+1 for error here, failing silently would be confusing and I'd be cursing at the computer failing to realise why I can't find my files where I expect them to be.